src/waveform/renderers/allshader/waveformrendererslipmode.cpp

```diff
@@ -0,0 +1,107 @@
+#include "waveform/renderers/allshader/waveformrendererslipmode.h"
+
+#include <QDomNode>
+#include <memory>
+
+#include "control/controlproxy.h"
+#include "waveform/renderers/waveformwidgetrenderer.h"
+#include "waveform/waveformwidgetfactory.h"
+#include "widget/wskincolor.h"
+
+namespace {
+
+constexpr int kBlinkingPeriodMillis = 1600;
+constexpr float positionArray[] = {-1.f, -1.f, 1.f, -1.f, -1.f, 1.f, 1.f, 1.f};
```

Comment:
This anonymous array is hard to understand. Can we improve on that, at least with a comment?